README.md

```diff
@@ -1,2 +1,166 @@
+- Country: United States
+- Organization: University of Illinois at Urbana-Champaign
+- Web page: http://www.matthewfeickert.com/
+- Corresponding Author: Yes
+
+## Author 2
+
+- First Name: Lukas
+- Last Name: Heinrich
+- Email: lukas.heinrich at-no-spam cern.ch
+- Country: Germany
+- Organization: CERN
+- Web page: http://www.lukasheinrich.com/
+- Corresponding Author: No
+
+## Author 3
+
+- First Name: Giordon
+- Last Name: Stark
+- Email: gstark at-no-spam cern.ch
+- Country: United States
+- Organization: University of California Santa Cruz Santa Cruz Institute for Particle Physics (UCSC SCIPP)
+- Web page: https://giordonstark.com/
+- Corresponding Author: No
+
+## Title and Abstract
+> The title and the abstract should be entered as plain text, they should not contain HTML elements.
+
+### Title
+
+pyhf: a pure Python statistical fitting library with tensors and autograd
+
+### Abstract
+
+> c.f. the [Planning for your proposal submission?](https://github.com/matthewfeickert/SciPy2020-Proposal/blob/master/info_page.md#planning-for-your-proposal-submission) information page
+
+> Your placement in the program will be based on reviews of your abstract.
+This should be a roughly 500 word outline of your presentation.
+This outline should concisely describe software of interest to the SciPy community, tools or techniques for more effective computing, or how scientific Python was applied to solve a research problem.
+A traditional background/motivation, methods, results, and conclusion structure is encouraged but not required.
+Links to project websites, source code repositories, figures, full papers, and evidence of public speaking ability are encouraged.
+
+#### Introduction and Motivation
+
+In experimental high energy physics (HEP), statistical fitting tools have been implemented almost entirely in C++.
+One common example used by experimentalists at CERN uses the "HistFactory" p.d.f. template.
+Prominently used in the 2012 discovery of the Higgs boson by the ATLAS and CMS collaborations, the p.d.f. template has only existed within the C++ based ROOT framework developed and maintained by CERN.
+pyhf is a pure-Python implementation of that statistical model for multi-bin histogram-based analysis with asymptotic interval estimation, and part of the Scikit-HEP project (https://scikit-hep.org/) ecosystem.
+pyhf supports modern computational graph libraries as computational backends in order to make use of features such as auto-differentiation and GPU acceleration.
+Additionally, the statistical models are defined in an expressive JSON schema, readily enabling preservation and distribution through services such as the Durham High-Energy Physics Database (HEPData) (https://www.hepdata.net/).
+
+#### Performance gain through tensorization
+
+Through adoption of open source "tensor" computational Python libraries, pyhf decreases the abstractions between a physicist analyst and the statistical modeling without sacrificing computational speed.
```

Comment:
```suggestion
Through adoption of open source "tensor" computational Python libraries, pyhf decreases the abstractions between a physics analysis and the statistical modeling without sacrificing computational speed.
```

Reply:
I really do mean an analyst here &mdash; a person &mdash; not an analysis.